### PR TITLE
Add command to create and start a  confidential instance

### PR DESCRIPTION
**Pull request by olethanh**
* Add --confidential argument to `aleph instance create`
* Add `aleph  instance confidential-init-session ` command to init the confidential sessions (get platform certificate, generate session certificate and upload them)
* Add `aleph  instance confidential-start ` command to fetch the measurement from the server  and start the VM (get measurements from server, validate them again locally calculated one and send the secret to the server)

I tested command separately against staging pyaleph and aleph-vm but the whole flow need to be thoroughly tested.

### CODE REVIEW

File: src/aleph_client/commands/instance/__init__.py [hunk 1]
```diff
@@ -168,10 +206,12 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
...
-    # For PAYG, the user select directly the node on which to run on
-    #  Use have to make the payment stream separately for no
+    # For PAYG or confidential, the user select directly the node on which to run on
+    # For PAYG User have to make the payment stream separately
+    # For now, we allow hold for confidential, but the user still has to choose on which CRN to run.
     reward_address = None
-    if not hold:
+    crn = None
+    if not hold or confidential:
         crn = None
```
Comment (Psycojoker) [inline src/aleph_client/commands/instance/__init__.py:215]: You can remove this line I think since it's already there line 213

**Review by Psycojoker — COMMENTED** [review]
Just some general comments on the code.

File: src/aleph_client/commands/confidential/__init__.py
[the remark this answers is not in the document]
Comment (olethanh) [reply src/aleph_client/commands/confidential/__init__.py:1]: yep error

File: src/aleph_client/commands/instance/__init__.py [hunk 2]
```diff
@@ -533,3 +532,123 @@ async def stop(
...
+    if firmware_file:
+        firmware_path = Path(firmware_file)
+        if not firmware_path.exists():
+            raise Exception("Firmware path does not exist")
+        firmware_hash = calculate_firmware_hash(firmware_path)
+        logger.info(f"Calculated Firmware hash: {firmware_hash}")
+    logger.info(sev_data)
+    valid = await client.validate_measure(sev_data, tik_path, firmware_hash=firmware_hash)
+    if not valid:
+        echo("Could not validate authenticity of the VM")
```
[the remark this answers is not in the document]
Comment (olethanh) [reply src/aleph_client/commands/instance/__init__.py:641]: I will put a more informative message. More debug info mean modifying the sdk and making a new release so will put that for later

**Review by Psycojoker — APPROVED** [review]

**Comment by Psycojoker** [comment]
@olethanh you can squash & merge whenever you want for me

**Review by nesitor — APPROVED** [review]

**Pull request merged**